packages/nodejs/src/extension_wrapper.ts

```diff
@@ -6,12 +6,15 @@ try {
   mod = require("../build/Release/extension.node") as ExtensionWrapper
   mod.isLoaded = true
 } catch (e) {
+  console.warn(
```

Comment:
```suggestion
  console.error(
```

It can probably be an error, because it's caused by an error. As long as it won't crash the program.

Reply:
Done!